src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2581,13 +2581,13 @@ class DropViewStatementSegment(BaseSegment):
 
 @ansi_dialect.segment()
 class DropUserStatementSegment(BaseSegment):
-    """A `DROP` statement for role, user, or group."""
```

Comment:
We don't need to introduce ROLE/GROUP to ANSI in this PR so removing rather than splitting out